packages/language-server/server/completion/completion_resolve.js

```diff
@@ -0,0 +1,9 @@
+
+
+function onCompletionResolve(item) {
+  item.detail = 'Type';
+  item.documentation = 'JavaScript documentation';
+  return item;
```

Comment:
And there:
```
return {
  ...item,
  detail: "Type",
 documentation: "JavaScript Documentatnion"
};
```
I really like immutable style of programming, so, try to keep the style in our projects 😄

Reply:
Okay, I will try)